jenkins/run-server-func-tests

```diff
@@ -68,14 +68,14 @@ if [[ -z ${1} ]]; then
     exit ${exit_status}
 fi
 
-podman pod stop ${PB_POD_NAME}
+# Remove the server container
+podman stop ${PB_SERVER_CONTAINER_NAME}
```

Comment:
I am _so_ sorry to have to ask this (I'm a very bad reviewer today), but...

Don't we need a corresponding addition to the `cleanup` function at line 12?

Reply:
> I am _so_ sorry to have to ask this (I'm a very bad reviewer today), but...
No, It's always good to catch the potential pitfalls.

Yeah, I should also stop the container in the cleanup for sure.

Reply:
But you not only left the ordering problem here but used the same ordering in cleanup? (Where, arguably, it might be slightly less important, but still... 😆 )

Reply:
Ah, I am hastily making all these changes because I think this one's almost ready and I have based my functional test changes on this PR. However, I have to push another update I think to revert the `PB_ORG_NAME` related change I made in `runlocal` and `pipeline.gy` files based on Peter's last comments on that conversation [comment](https://github.com/distributed-system-analysis/pbench/pull/3216#discussion_r1094979720)